lucene/backward-codecs/src/test/org/apache/lucene/backward_codecs/lucene60/TestLucene60FieldInfosFormat.java

```diff
@@ -14,17 +14,15 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.lucene.codecs.lucene50;
```

Comment:
Hmm, why did this previously live under `lucene50` sub-package if it was `lucene60` format?

Reply:
This indeed seems like a mistake.